Lib/sqlite3/dump.py

```diff
@@ -14,6 +14,11 @@ def _quote_name(name):
 def _quote_value(value):
     return "'{0}'".format(value.replace("'", "''"))
 
+def _force_decode(bs, *args, **kwargs):
+    try:
+        return bs.decode(*args, **kwargs)
+    except UnicodeDecodeError:
+        return "".join(chr(c) for c in bs)
```

Comment:
This is slightly faster:

```suggestion
        return "".join([chr(c) for c in bs])
```

Reply:
I pretty sure we'll never get a `ValueError` from the `chr` conversion.

Reply:
Yes, `chr()` should be good for any single byte I think.